LeetCode-Scrapper/ques.py

```diff
@@ -0,0 +1,119 @@
+from selenium import webdriver
+import os
+options = webdriver.ChromeOptions()
+options.add_argument("--headless")
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from selenium.common.exceptions import TimeoutException
+from fpdf import FPDF
+
+
+capa = DesiredCapabilities.CHROME
+capa["pageLoadStrategy"] = "none"
+
+# Input for chrome driver path
+chromeDriverPath=  "C:\chromedriver_win32\chromedriver.exe";
```

Comment:
When I say take input for chrome driver path, I mean to say let the user enter the path of their chrome driver via the terminal. use input function